logistic_order_multicurrency/model/sale_order.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Copyright 2015 Camptocamp SA
+#    Author: Yannick Vaucher
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+from openerp import models, fields, api
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.one
+    @api.depends('amount_total',
+                 'currency_id',
+                 'company_id.currency_id')
+    def _compute_prices_in_company_currency(self):
+        """ Compute amount_total in company currency
+
+        Date for conversion is today
```

Comment:
are we sure we want to use today, always? I would be tempted have this depend on the state of the SO: maybe use today in draft mode, and the day of confirmation of the SO after. Or maybe we want today until the SO is paid?

@jgrandguillaume: opinion?